### PR TITLE
polished the language in README.zh-CN.md & sync with English one

### PR DESCRIPTION
**Pull request by FrozenArcher**
Plus, what is the "HEAD of sway and wlroots"?

### CODE REVIEW

**Comment by RPigott** [comment]
HEAD means the latest commit in the master branch. Sway and wlroots are developed together, so the latest sway should be built with the latest wlroots.

**Comment by emersion** [comment]
@SuccessGo @3TUSK, would you have time to have a look at this new translation?

File: README.zh-CN.md [hunk 1]
```diff
@@ -1,28 +1,23 @@
...
-Sway 在很多发行版中可用. 尝试在你的发行版中安装 "sway" 包.
-如果这不可用, 请到 [此 wiki 页](https://github.com/swaywm/sway/wiki/Unsupported-packages)
-检查针对你的发行版关于安装的信息.
+Sway 在很多发行版中可用。请尝试在你的发行版中安装 `sway` 。
 
-如果你有兴趣给你的发行版打包 sway, 停下来到 IRC 频道或者发邮件至 sir@cmpwn.com 获取建议.
+### 从源码编译
 
-### 从源代码编译
+如果想要构建sway和wlroots用以测试和开发，请查看 [此wiki页面](https://github.com/swaywm/sway/wiki/Development-Setup)
```
Comment (3TUSK) [inline README.zh-CN.md:18]:
```suggestion
如果想要构建最新版sway和wlroots用以测试和开发，请查看 [此wiki页面](https://github.com/swaywm/sway/wiki/Development-Setup)
```

[`HEAD` 指当前 git 仓库的当前分支的最新提交](https://stackoverflow.com/questions/2304087/what-is-head-in-git)，上面有人已经解释过了。

**Comment by FrozenArcher** [comment]
@emersion "Review required"

**Comment by emersion** [comment]
@3TUSK, are you happy with this version?

@FrozenArcher, can you rebase and squash all of these commits into a single one?

**Comment by FrozenArcher** [comment]
@emersion can I squash them directly in github? I'm new to the workflows -_-b
(I followed some steps in stackoverflow, but I don't know how to push them. since github has removed password authentication, I added a ssh key and pushed it to a new ssh remote but nothing was uploaded.)
Nothing has changed since 3TUSK's suggestion was committed, so he will certainly be happy with this version ;-)

**Comment by FrozenArcher** [comment]
Squashed via github desktop. @emersion @3TUSK

**Comment by emersion** [comment]
@3TUSK, gentle ping

**Review by 3TUSK — APPROVED** [review]

**Review by emersion — COMMENTED** [review]
Thank you both!

**Pull request merged**